tests/deployment/test_client_server.py

```diff
@@ -85,6 +82,10 @@ def test_client_server_sklearn(
 ):
     """Tests the encrypt decrypt api."""
 
+    # if get_model_name(model_class) == "KNeighborsClassifier":
```

Comment:
is this supposed to be uncommented ?

Reply:
if so please add `FIXME:` before the link and most of all put "https://github.com/zama-ai/concrete-ml-internal/issues/4014" instead of #4019 !

Reply:
Yes it wasn't supposed to be commented.

I replaced by 4014, thanks @RomanBredehoft